gradle/libs.versions.toml

```diff
@@ -59,6 +60,7 @@ androidx-datastore = { module = "androidx.datastore:datastore", version.ref = "a
 androidx-espresso-core = { module = "androidx.test.espresso:espresso-core", version.ref = "androidxTestEspresso" }
 androidx-graphics-core = { module = "androidx.graphics:graphics-core", version.ref = "androidxGraphicsCore" }
 androidx-junit = { module = "androidx.test.ext:junit", version.ref = "androidxTestJunit" }
+androidx-lifecycle-livedata = { module = "androidx.lifecycle:lifecycle-livedata-ktx", version.ref = "androidxLifecycle" }
```

Comment:
I can't found anywhere useing LiveData, is there I am missing point?

Reply:
It is used by `cameraInfo.torchState.asFlow()` in `CameraXCameraUseCase.kt`.